server/tsconfig.json

```diff
@@ -2,12 +2,24 @@
 	"compilerOptions": {
 		"target": "es6",
 		"module": "commonjs",
+		"moduleResolution": "node",
+		"noImplicitAny": false,
```

Comment:
We want to guard against using `any` since the whole idea behind Typescript is to type what we're using. This will keep us safe from unknowns and make the shape of what we're working with more manageable and known